packages/tupaia-web/src/features/DashboardItem/DashboardInfoButton.tsx

```diff
@@ -0,0 +1,67 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
+ */
+
+import React from 'react';
+import styled from 'styled-components';
+import { ViewConfig } from '@tupaia/types';
+import MuiInfoIcon from '@material-ui/icons/Info';
+import { MOBILE_BREAKPOINT } from '../../constants';
+
+const OpenDashboardInfoButton = styled.div`
+  text-transform: none;
+  font-weight: ${({ theme }) => theme.typography.fontWeightRegular};
+  cursor: pointer;
+  color: ${({ theme }) => theme.palette.common.white};
+  border-color: ${({ theme }) => theme.palette.common.white};
+  padding: 0.3rem;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  gap: 0.3rem;
+  margin-top: 1rem;
+  @media screen and (min-width: ${MOBILE_BREAKPOINT}) {
+    .MuiButton-label {
+      display: flex;
+      flex-direction: column;
+      justify-content: center;
+      align-items: center;
+    }
+    position: absolute;
+    width: 100%;
+    height: 100%;
+    background-color: ${({ theme }) => theme.palette.common.black};
+    border: none;
+    top: 0;
+    left: 0;
+    opacity: 0;
+    margin-top: 0;
+    &:hover,
+    &:focus-visible {
+      opacity: 0.9;
+      background-color: ${({ theme }) => theme.palette.common.black};
+    }
+  }
+`;
+
+const DashboardInfoButtonText = styled.span`
+  font-size: 1rem;
+`;
+
+interface DashboardInfoButtonProps {
+  viewType?: ViewConfig['viewType'];
+  infoText: string | undefined;
+}
+
+export const DashboardInfoButton = ({ viewType, infoText }: DashboardInfoButtonProps) => {
+  const content = <DashboardInfoButtonText>{infoText}</DashboardInfoButtonText>;
```

Comment:
Can we get this to return `null` if `infoText` is not set? It also looks like this is only visible for `SingleValue` view types, and only when `periodGranularity` is not set in the config. So perhaps instead of rendering this for all view types, just add to the single value one. Unless Andrew knows of any other types? 
If it's just the one type, then it doesn't even need to be a separate file, it can just be a styled component inside the SingleValue file